skrub/_drop_null.py

```diff
@@ -0,0 +1,54 @@
+# drop columns that contain all null values
+from sklearn.utils.validation import check_is_fitted
+
+from . import _dataframe as sbd
+from ._on_each_column import SingleColumnTransformer
+
+__all__ = ["DropNullColumn"]
+
+
+class DropNullColumn(SingleColumnTransformer):
+    """Drop a single column if it contains only null values."""
+
+    def __init__(self):
+        super().__init__()
+        self._is_fitted = False
```

Comment:
```suggestion
```

to follow the scikit-learn requirements, estimators must not do anything in their __init__ except store the parameters. here you don't have any parameters to store so you don't need to define __init__